changes/bug24953

```diff
@@ -0,0 +1,3 @@
+  o Minor bugfixes (fallback scripts):
+    - In updateFallbackDirs.py, say "fallback list" instead of "whitelist".
```

Comment:
```suggestion
    - In updateFallbackDirs.py, call the filter file a "fallback list" instead of a "whitelist" in check_existing mode.
```

Reply:
Fixed in bf749490432c4e82fc415434fa9f380225303b71.